tsconfig.json

```diff
@@ -22,5 +22,9 @@
     "target": "esnext",
     "types": []
   },
-  "files": ["node_modules/typescript/lib/lib.esnext.d.ts", "js/main.ts"]
+  "files": [
+    "node_modules/typescript/lib/lib.esnext.d.ts",
+    "node_modules/typescript/lib/lib.es2017.sharedmemory.d.ts",
```

Comment:
Is this line necessary? Shouldn't it be included from `lib.esnext.d.ts` ?

Reply:
Removed this patch as it didn't seem to fully fix the problem it was supposed to fix.